sdks/unity/model/SpecHealth.cs

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 Google LLC
+// All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Agones.Model
+{
+    /// <summary>
+    /// SpecHealth
+    /// </summary>
+    public class SpecHealth : IEquatable<SpecHealth>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SpecHealth" /> class.
+        /// </summary>
+        public SpecHealth(IReadOnlyDictionary<string, object> data)
+        {
+            this.Disabled = data.TryGetValue("disabled", out var disabled) && bool.Parse((string) disabled);
+            this.PeriodSeconds = (Int64) data["period_seconds"];
```

Comment:
Can we use Int64.Parse for these as well?

Reply:
This is an actual `Int64` stored in the map, so there is no string to parse.